tutorials/index.md

```diff
@@ -1,5 +1,13 @@
 # Tutorials
 
+These tutorials will help you get familiar with Chainblocks.
+
+## Setting up the workspace
+
+We will be using <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a>, although any text editor could work.
```

Comment:
We try to avoid HTML inside markdown document. It's ok if the link doesn't open in a new tab or window.

Keeping "pure" markdown will allow using the same document from within the game UI using a simple markdown parser, instead of requiring a HTML renderer.

Reply:
Reverted to markdown link tag in latest commit.